### PR TITLE
Display the languages menu in the admin depending on the section

### PR DESCRIPTION
**Pull request by paolina92**

### CODE REVIEW

File: assembl/static2/js/app/reducers/adminReducer/index.js [hunk 1]
```diff
@@ -198,6 +198,17 @@ export const discussionLanguagePreferencesHasChanged: DiscussionLanguagePreferen
   }
 };
 
+type ToggleLanguageMenu = boolean;
+type ToggleLanguageMenuReducer = (ToggleLanguageMenu, ReduxAction<Action>) => ToggleLanguageMenu;
+export const toggleLanguageMenu: ToggleLanguageMenuReducer = (state = false, action) => {
+  switch (action.type) {
+  case 'TOGGLE_LANGUAGE_MENU':
+    return action.state;
```
Comment (cedricmessiant) [inline assembl/static2/js/app/reducers/adminReducer/index.js]: if it is a toggle action, I think you should handle the fact that it toggles the value here (so no value in the action and `return !state`). If you need to be able to choose the value, I would rename the action `UPDATE_LANGUAGE_MENU_VISIBILITY` or something)

File: assembl/static2/js/app/reducers/adminReducer/index.js [hunk 2]
```diff
@@ -198,6 +198,17 @@ export const discussionLanguagePreferencesHasChanged: DiscussionLanguagePreferen
   }
 };
 
+type ToggleLanguageMenu = boolean;
+type ToggleLanguageMenuReducer = (ToggleLanguageMenu, ReduxAction<Action>) => ToggleLanguageMenu;
+export const toggleLanguageMenu: ToggleLanguageMenuReducer = (state = false, action) => {
```
Comment (cedricmessiant) [inline assembl/static2/js/app/reducers/adminReducer/index.js]: I think it is really a strange name for a reducer, maybe use something like `languageMenuIsVisible` or `displayLanguageMenu`

Comment (paolina92) [reply assembl/static2/js/app/reducers/adminReducer/index.js]: I agree. Let's do that...

File: assembl/static2/js/app/reducers/adminReducer/index.js (hunk 1 above)
Comment (paolina92) [reply assembl/static2/js/app/reducers/adminReducer/index.js]: ok

**Pull request merged**